rustv1/examples/s3/src/bin/select-object-content.rs

```diff
@@ -7,11 +7,12 @@
 
 use aws_config::meta::region::RegionProviderChain;
 use aws_sdk_s3::types::{
-    CompressionType, CsvInput, CsvOutput, ExpressionType, FileHeaderInfo, InputSerialization,
+    CompressionType, CsvInput, ExpressionType, FileHeaderInfo, InputSerialization, JsonOutput,
     OutputSerialization, SelectObjectContentEventStream,
 };
 use aws_sdk_s3::{config::Region, meta::PKG_VERSION, Client, Error};
 use clap::Parser;
+use serde::Deserialize;
```

Comment:
Looks like you need to pull in `serde` to the Cargo.toml (which I assume came from the other PR?)